### PR TITLE
Add the ability to configure text to pick up an item

### PR DESCRIPTION
**Pull request by PerseweratywneImponderabilia**

### CODE REVIEW

File: item.inc [hunk 1]
```diff
@@ -74,6 +74,10 @@
     #define ITEM_GAMETEXT_INTERACT "~k~~VEHICLE_ENTER_EXIT~"
 #endif
 
+#if !defined ITEM_GAMETEXT_PICKUP
+    #define ITEM_GAMETEXT_PIKCUP "Press "ITEM_GAMETEXT_INTERACT" to pick up"
```
Comment (Southclaws) [inline item.inc]:
```suggestion
    #define ITEM_GAMETEXT_PICKUP "Press "ITEM_GAMETEXT_INTERACT" to pick up"
```

**Review by Southclaws — CHANGES_REQUESTED** [review]

File: item.inc (hunk 1 above)
Comment (PerseweratywneImponderabilia) [reply item.inc]: Is there any reason for that? By using the `ITEM_GAMETEXT_INTERACT` definition we don't have to edit the key in 2 places.
Like let's assume I'd like to change the key from `ENTER` to `N` but that's about it. In this case all I have to do so is change the `ITEM_GAMETEXT_INTERACT` definition and the pick-up text will get updated as well without needed any further changes

Comment (Southclaws) [reply item.inc]: Oh yeah sorry misread the code this morning, with the typo fixed it should be good to merge!

I forgot about the constant embedding of Pawn! So `"ITEM_GAMETEXT_INTERACT"` looked odd.

Comment (PerseweratywneImponderabilia) [reply item.inc]: Oh, the typo! Thanks. Should be all right now. :)

**Review by Southclaws — APPROVED** [review]

**Pull request merged**